2020-InfiniteRecharge/src/main/java/frc/robot/Robot.java

```diff
@@ -33,11 +35,12 @@
   private static final String kCustomAuto = "My Auto";
   private String m_autoSelected;
   private final SendableChooser<String> m_chooser = new SendableChooser<>();
+  private IMU imu;
+  public OperatorInterface oi;
   private CANSparkMax spark1;
   private CANSparkMax spark2;
-  private Chassis driveTrain;
-  private OperatorInterface oi;
-
+  //private Chassis driveTrain;
```

Comment:
It's time to start using feature flags so that we're not commenting out each other's subsystems.